components/menu/OverflowMenu/stories-style.module.scss

```diff
@@ -1,5 +1,11 @@
 @import "../../../styles/index";
 
```

Comment:
I meant removing the style for this entirely. Since we're using `children`, the app using MDL can supply the styling till we have this rule locked down.

Reply:
Done